cdk/PgStacInfra.ts

```diff
@@ -54,6 +56,12 @@ export class PgStacInfra extends Stack {
       subnetSelection: apiSubnetSelection,
     });
 
+
+    stacApiLambda.stacApiLambdaFunction.addPermission('ApiGatewayInvoke', {
```

Comment:
Naming got a little redundant with the definition in cdk-pgstac but not really sure what else it would be so maybe it's fine? 🤷🏼‍♂️

Reply:
yea not sure what else that could be, perhaps `stacApiLambdaPythonFunction`....